seacrowd/sea_datasets/wikiann/wikiann.py

```diff
@@ -71,15 +70,20 @@
 Minangkabau	min	min
 Sundanese	su	sun
 Acehnese	ace	ace
-Malay	ms	mly
+Malay	ms	zlm
 Banyumasan	map-bms	map-bms
+Myanmar my mya
+Tagalog tl tgl
+Thailand th tha
+Vietnam vi vie
+Khmer km khm
 
 
 """
 
 _HOMEPAGE = "https://github.com/afshinrahimi/mmner"
 
-_LICENSE = "Apache-2.0 license"
+_LICENSE = Licenses.APACHE_2_0
```

Comment:
you seems to forgot referencing its value
```suggestion
_LICENSE = Licenses.APACHE_2_0.value
```

Reply:
adjusted, thank you!